conda_devenv/__init__.py

```diff
@@ -1,3 +1,4 @@
-from ._version import version
-
-__version__ = version
+try:
+    from ._version import version as __version__
+except ModuleNotFoundError:
+    __version__ = "0.0.0"
```

Comment:
Just a question.
Did you stumble into some problem with this?
Like testing the changes (obviously, in an unreleased) version?

Reply:
Yeah, I was having some problems to run all the tests, specially those which test the ``conda-devenv`` version 
I can change if you want to, I just thought that it would be nice to able to run all tests without a release in place

Reply:
I agree with you about being able to run the tests.